torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -203,7 +206,7 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
                 and node.target is torch.ops.aten._local_scalar_dense.default
             ):
                 dtype = node.args[0].meta["val"].dtype
-                if dtype != torch.float64:
+                if not is_float_dtype(dtype):
```

Comment:
If the dtype is float32 or narrower, do we have to explicitly do a conversion here?